main.tf

```diff
@@ -288,6 +288,27 @@ resource "juju_application" "traefik-public" {
   units  = var.ingress-scale
 }
 
+resource "juju_application" "traefik-rgw" {
```

Comment:
This traefik should only be enabled when there's a storage role. There there's should be a condition on its activation.

Reply:
Done